framework/include/transfers/MultiAppProjectionTransfer.h

```diff
@@ -57,6 +59,8 @@ class MultiAppProjectionTransfer : public MultiAppTransfer
   friend void assemble_l2_from(EquationSystems & es, const std::string & system_name);
   friend void assemble_l2_to(EquationSystems & es, const std::string & system_name);
 
+  NumericVector<Number> * _serialized_master_solution;
```

Comment:
It's needed in the assemble functions which we can't pass extra arguments into (as best I can tell)